content/en/docs/concepts/architecture/nodes.md

```diff
@@ -158,9 +158,9 @@ The `conditions` field describes the status of all `Running` nodes. Examples of
 {{< /table >}}
 
 {{< note >}}
-If you use command-line tools to print details of a cordoned Node, the Condition includes
-`SchedulingDisabled`. `SchedulingDisabled` is not a Condition in the Kubernetes API; instead,
-cordoned nodes are marked Unschedulable in their spec.
+If you use command-line tools to print details of a cordoned node, the condition includes
+`SchedulingDisabled`. `SchedulingDisabled` is not a condition in the Kubernetes API; instead,
```

Comment:
Here "condition in the Kubernetes API" refers to the `Condition` object in the Kubernetes API?